lib/ObjWriter/.nuget/Linux/Microsoft.Dotnet.ObjectWriter.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.DotNet.ObjectWriter</id>
+    <version>1.0.0-prerelease</version>
+    <title>Microsoft .NET ObjectWriter</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>https://github.com/dotnet/corert</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Provides the toolchain to compile managed code to native.</description>
```

Comment:
"Provides object writer to the managed to native code-generator"
Let me know if there is other suggestion.